lzy/graph-executor-2/src/main/java/ai/lzy/graph/model/Task.java

```diff
@@ -0,0 +1,28 @@
+package ai.lzy.graph.model;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+import java.util.List;
+
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
+@JsonSerialize
+@JsonDeserialize
+public record Task(
```

Comment:
is it about state of a task?
if so, let's rename it to `TaskState` (and same for other state objects)